features/view_all_companies.feature

```diff
@@ -162,6 +165,7 @@ Feature: Visitor sees all companies
     Given the date is set to "2017-10-01"
     Given I am Logged out
     And I am on the "landing" page
+    And show me the page
```

Comment:
Is this for debugging?  (probably don't want it in production code).

Reply:
doh!